docs/advanced_reference/commands/autonomy_deploy.md

```diff
@@ -149,14 +144,17 @@ autonomy deploy from-token [OPTIONS] TOKEN_ID KEYS_FILE
 `--aev`
 :   Apply environment variable when loading service config.
 
+`--use-ethereum`
+: Use ethereum to resolve the token id.
+
 `--use-goerli`
-:   Use Görli chain to resolve the `TOKEN_ID`.
+: Use goerli to resolve the token id.
```

Comment:
What is "gorli" in lowercase in this sentence? Please be more specific, otherwise the sentence is confusing.

Reply:
Also, "token id" when the intro of this command uses `TOKEN_ID`. Should choose one naming and be consistent with it.

Reply:
Note that the intro of this document is out of sync with the new options. When making changes in the CLI, please ensure the whole command documentation is up to date, otherwise these out-of-sync errors in documentation will be very difficult to identify.

Reply:
This PR is to add simple documentation, you can create a ticket to polish this documentation once the base PR is merged